.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Description: 
+Summarize the changes you're submitting in a few sentences, including Jira ticket ATL-xxxx if applicable.
+Link to any discussion, related issues and bug reports to give the context to help the reviewer understand the PR.
+
+### Alternatives Considered (optional): 
+Link to existing ADR (Architecture Decision Record), if any. If relevant, describe other approaches explored and the selected approach. Documenting why the methods were not selected will create a knowledge base for future reference, helping prevent others from revisiting less optimal ideas.
+
+### Checklist: 
+- [] My PR follows the contribution guidelines of this project
```

Comment:
It would be nice to have a link to the guidelines

Can't this just be a text that the user deletes after reading.
Why do this a Checklist. Is just one more item for the reviewer to read.

Reply:
@FabioPinheiro , good idea: I do not see any contribution guideline in mediator repo by the way. Let's add one and we will then link it here.

Reply:
@FabioPinheiro , I have added your comment to take into account in the issue #254. If ok, please resolve the conversation.